formats/placeholder/frontend/examples/organisation.json

```diff
@@ -0,0 +1,36 @@
+{
+  "base_path": "/government/organisations/bona-vacantia",
+  "content_id": "c9109663-0128-4db4-b513-a9cdfb691034",
+  "title": "Bona Vacantia",
+  "format": "placeholder_organisation",
+  "document_type": "organisation",
+  "schema_name": "placeholder",
```

Comment:
To clarify this, I think the `format` is ok, but I don't have a huge amount of context yet on the `document_type`/`schema_name` split. Does this look ok to you @danielroseman?